lib/conversation/replies.js

```diff
@@ -0,0 +1,148 @@
+'use strict';
+
+/* eslint-disable no-param-reassign */
+
+const logger = require('winston');
+
+const Command = require('./command');
+const actions = require('./actions');
+const helpers = require('../helpers');
+const stathat = require('../stathat');
+const BotRequest = require('../../app/models/BotRequest');
+
+function customMsgDecorator(fn) {
+  return (args) => {
+    fn(args);
+    return helpers.sendResponse(args.res, 200, args.text);
+  };
+}
+
+function renderEndConversationMsg(fn) {
+  return (type, args) => helpers.renderMessageForMessageType(args.req, type)
+      .then((replyMessage) => {
+        args.text = replyMessage;
```

Comment:
I'm sending the `req`, `res`, and `replyText` inside an `args` object so I don't have to bother with the variadic nature of the replies. The point here is that each reply gets all the "ingredients" necessary to return a new function (a command) that the dispatcher executes.

I could have also approached this by sending arguments as needed:
- `replyDispatcher.execute(replies.legacyBugErrorOcurred(req, res, err))`
Instead of:
- `replyDispatcher.execute(replies.legacyBugErrorOcurred({ req, res, error: err }))`
Then define each reply and decorators with the needed parameters, etc. Which I think It would make the code more complex:
    ```
    module.exports.legacyBugErrorOcurred = function legacyBugErrorOcurred(req, res, error) {
      return new Command(
        renderEndConversationWithErrorMsg(actions.endConversation),
         ['error_occurred', req, res, error]);
    };
```